.github/workflows/update-snapcraft.yaml

```diff
@@ -0,0 +1,44 @@
+on:
+  workflow_call:
+    inputs:
+      openstack-release:
+        description: OpenStack release
+        type: string
+        required: true
+
+jobs:
+  update-snapcraft:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: stable/${{ inputs.openstack-release }}
+      - name: Install Workflow Dependencies
+        run: pip install feedparser pygit2 pyyaml semver
```

Comment:
these are the requirements for your update-snapcraft script. Wouldn't it be easier to hold them in a `requirements.txt` file for ease of debugging (i.e. when you may want to step through `update-snapcraft` from your local machine)?

Reply:
introduced `requirements-dev.txt` in the update.